src/pages/print-issues.tsx

```diff
@@ -41,10 +38,12 @@ const PrintIssues = ({
 }) => {
   return (
     <Layout>
-      <Wrapper>
-        <H1>Print Issues</H1>
-        <Subtitle>Checkout all our print issues... digitally</Subtitle>
-        <IssuesHolder>
+      <Container style={{ paddingTop: "50px" }}>
```

Comment:
`Container` is a stitches component so you can add/remove/change stylings to it with the [`css`](https://stitches.dev/docs/overriding-styles) prop